test/spec/relative-date-filter.js

```diff
@@ -36,7 +36,7 @@ describe('Filter: relativeDate', function() {
     expect(relativeDate('2013-09-06T12:00:00Z')).toEqual('yesterday')
     expect(relativeDate('2013-09-06T00:00:00Z')).toEqual('yesterday')
     expect(relativeDate('2013-09-05T22:59:59Z')).toEqual('2 days ago')
-    expect(relativeDate('2013-09-01')).toEqual('6 days ago')
+    expect(relativeDate('2013-09-01T12:00:00Z')).toEqual('6 days ago')
```

Comment:
I don't quite remember, but I think I was getting failed tests before I made this change.  Will look into it this weekend